src/shipping.js

```diff
@@ -42,7 +42,7 @@ export function getLogisticsInfoData(params) {
       : logisticsInfo[itemIndex].shippingEstimate
         ? logisticsInfo[itemIndex].shippingEstimate
         : undefined,
-    deliveryWindow: selectedSla ? selectedSla.deliveryWindow : undefined,
+    deliveryWindow: logisticsInfo[itemIndex].deliveryWindow,
```

Comment:
Just remember this is a breaking change and you'll need to generate a major. If someone is checking for `deliveryWindow` as `logisticInfo && logisticsInfo.deliveryWindow` will have a different result for now on.

Reply:
You're right! I think it shouldn't be a breaking change :( but I can't think of a solution to avoid this. Do you have any considerations?

Reply:
Also, do you have any special way to handle this major release? Is it ok to only release the new major and merge this to master?

Reply:
@arthurepc I think we should bring this discussion to front-end-chapter channel. We need to understand which projects do use this and how they would be affected.
Maybe you could explain a little bit better the problem you are facing so we can suggest another approach.
What do you think?

Reply:
Order and OrderForm have different formats, this won't work on an OrderForm object, explained more on the original Slack of this PR! :D